pkg/ebpf/config.go

```diff
@@ -18,6 +18,10 @@ type Config struct {
 	// CollectLocalDNS specifies whether the tracer should capture traffic for local DNS calls
 	CollectLocalDNS bool
 
+	// DNSSnooping specifies whether the tracer should enhance connection data with domain names by snooping DNS traffic
```

Comment:
nit: can we replace `snooping` with `inspection`

Reply:
makes sense 😄